.github/workflows/main.yml

```diff
@@ -132,6 +138,7 @@ jobs:
     - name: Build (CMake)
       run: cmake --build build/ --config Release --parallel --verbose
     - name: Run build-time tests
+      if: "runner.os != 'Windows'"
```

Comment:
The build-time tests are also not executed for SDL2_Image.
This is probably because of `PATH` issues.